src/output.jl

```diff
@@ -172,41 +172,32 @@ end
 """Checks output folders to determine a 4-digit run id number."""
 function get_run_id_path(S::ModelSetup)
 
-    @unpack output,outpath,get_id_mode = S.parameters
+    @unpack output,outpath,get_id_mode,path = S.parameters
 
     if output
-        runlist = filter(x->startswith(x,"run"),readdir(outpath))
-        existing_runs = [parse(Int,id[4:end]) for id in runlist]
+        
+        pattern = r"run_\d\d\d\d"               # run_???? in regex
+        runlist = filter(x->startswith(x,pattern),readdir(path))
+        runlist = filter(x->endswith(  x,pattern),runlist)
+        existing_runs = [parse(Int,id[5:end]) for id in runlist]
+
+        # get the run id from existing folders
         if length(existing_runs) == 0           # if no runfolder exists yet
-            runpath = joinpath(outpath,"run0000")
-            mkdir(runpath)
-            return 0,runpath
-        else                                    # create next folder
-            if get_id_mode == "fill"  # find the smallest gap in runfolders
```

Comment:
Note that you removed the `get_id_mode` options. This is fine with me, I only used them once in order to get 200 runs where some failed and I wanted to be able to fill in the gaps or predetermine the output id